File: QTD.SPA/src/app/_DtoModels/ILA_StudentEvaluation_Link/ILA_StudentEvaluation_LinkOptions.ts
export class ILA_StudentEvaluation_LinkOptions {
  iLAId!:any;
  studentEvalFormIDs:any[] = [];
  studentEvalAvailabilityID!:any;
  studentEvalAudienceID!:any;
  isAllQuestionMandatory!:boolean;
}


export class ILA_StudentEvaluation_LinkOption {
  studentEvalFormID!:any;
  studentEvalAvailabilityID!:any;
  studentEvalAudienceID!:any;
  isAllQuestionMandatory!:boolean;
  ilaStudenEvaluationLinkId!:any;
  isEvalRemove:boolean;
}

File: QTD2.Infrastructure/Model/ILA_StudentEvaluation_Link/ILA_StudentEvaluation_LinkOptions.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Infrastructure.Model.ILA_StudentEvaluation_Link
{
    public class ILA_StudentEvaluation_LinkOptions
    {
        public int ILAId { get; set; }

        public List<int> studentEvalFormIDs { get; set; }

        public int? studentEvalAvailabilityID { get; set; }

        public int? studentEvalAudienceID { get; set; }

        public bool isAllQuestionMandatory { get; set; }
    }
}
